scrapers/sc/events.py

```diff
@@ -83,6 +105,14 @@ def get_stream_id(self, onclick: str) -> str:
         return re.findall(r"live_stream\((\d+)|$", onclick)[0]
 
     def scrape(self, chamber=None, session=None):
+        if chamber:
+            yield self.scrape(chamber, session)
+        else:
+            yield self.scrape_single_chamber("legislature", session)
```

Comment:
Probably want `yield from` here so we aren't sending generators of generators back.